docs/tutorials/cargo_tests/baked/README.md

```diff
@@ -0,0 +1,9 @@
+# demo [![crates.io](https://img.shields.io/crates/v/demo)](https://crates.io/crates/demo)
```

Comment:
Does it make sense to generate READMEs for no-publish crates?

Reply:
Well, I made the README have a useful link to cargo.md, and it's more work to exclude these than it is to just make them partially useful.